torchgeo/trainers/segmentation.py

```diff
@@ -122,7 +122,23 @@ def configure_losses(self) -> None:
             )
 
     def configure_metrics(self) -> None:
-        """Initialize the performance metrics."""
+        """Initialize the performance metrics.
+
+        * Multiclass Pixel Accuracy: Ratio of correctly classified pixels. Uses
+          'micro' averaging. Higher values are better.
+        * Multiclass Jaccard Index (IoU): Per-pixel overlap between predicted and
+          actual segments.Uses 'macro' averaging. Higher values are better.
+
+        Uses 'micro' averaging, aggregating pixel predictions across classes. Each
+        pixel is weighted equally, favoring classes with more pixels. This may skew
+        performance metrics towards majority classes due to class imbalance.
```

Comment:
We don't have this for any other trainer and it's kinda covered in the note below, maybe we can remove it?

Reply:
Agreed, I think the note below + the updated description covers it all. I'm afk for the day but feel free to remove it if you're inclined.